Problem with Ceres Solver library

Hello, 

I had a problem when compiling ceres solver with your toolchain.

I suggest commenting the following lines in the toolchain file: 
```
set(CMAKE_CXX_COMPILER_FORCED TRUE)
set(CMAKE_CXX_COMPILER_WORKS TRUE)
set(CMAKE_C_COMPILER_FORCED TRUE)
set(CMAKE_C_COMPILER_WORKS TRUE)
```

The problem due to those lines is described in the following page:
[https://ceres-solver-review.googlesource.com/c/ceres-solver/+/13660/1/cmake/iOS.cmake#229](https://ceres-solver-review.googlesource.com/c/ceres-solver/+/13660/1/cmake/iOS.cmake#229)

**CMake version: 3.12.3**

While building [EntityX](https://github.com/alecthomas/entityx) iOS library generates following error.

    CMake Error at entityx/CMakeLists.txt:146 (target_compile_features):
    target_compile_features no known features for CXX compiler
    "AppleClang"
    version 10.0.0.10001145.

Consider this fix on [https://github.com/ceres-solver/ceres-solver/commit/7582834a7ec2838163a215372cfe72484a3a47a2](https://github.com/ceres-solver/ceres-solver/commit/7582834a7ec2838163a215372cfe72484a3a47a2) solve the problem above.

--- a/ios.toolchain.cmake
+++ b/ios.toolchain.cmake
@@ -309,11 +309,6 @@ set(CMAKE_OSX_DEPLOYMENT_TARGET "" CACHE STRING
 set(CMAKE_OSX_ARCHITECTURES ${IOS_ARCH} CACHE STRING "Build architecture for iOS")
 # Change the type of target generated for try_compile() so it'll work when cross-compiling
 set(CMAKE_TRY_COMPILE_TARGET_TYPE STATIC_LIBRARY)
-# Skip the platform compiler checks for cross compiling.
-set(CMAKE_CXX_COMPILER_FORCED TRUE)
-set(CMAKE_CXX_COMPILER_WORKS TRUE)
-set(CMAKE_C_COMPILER_FORCED TRUE)
-set(CMAKE_C_COMPILER_WORKS TRUE)
 # All iOS/Darwin specific settings - some may be redundant.
 set(CMAKE_SHARED_LIBRARY_PREFIX "lib")
 set(CMAKE_SHARED_LIBRARY_SUFFIX ".dylib")

